src/middlewared/middlewared/plugins/hardware/cpu.py

```diff
@@ -1,55 +1,68 @@
-from pathlib import Path
+import enum
+import os
 
 from middlewared.service import Service
 from middlewared.service_exception import ValidationError
-from middlewared.schema import accepts, returns, Dict, Str
 from middlewared.utils.functools_ import cache
 
 
-class HardwareCpuService(Service):
+class SysfsCpuEnum(enum.StrEnum):
```

Comment:
Why does this have to be an enum? You have to add `.value` every time you access a constant, but that’s it

Reply:
Which extra memory? If you create just
```
class SysfsCpuEnum:
    BASE = "/sys/devices/system/cpu/"
```
You'll be able to write `SysfsCpuEnum.BASE` instead of `SysfsCpuEnum.BASE.value`

Reply:
fixed it.